jekyll/_cci2/hello-world-windows.md

```diff
@@ -336,10 +336,19 @@ Above, we start by declaring that we will use version `2.1` of CircleCI, giving
 
 ```yaml
 orbs:
-  win: circleci/windows@2.2.0
+  win: circleci/windows@2.4.0
```

Comment:
Is there a reason this one didn't get updated to 4.1?

Reply:
Really only because the [example app](https://github.com/CircleCI-Public/circleci-demo-windows/blob/master/.circleci/config.yml) it refers to is still using the older version of the orb. I thought it might get confusing to not have the versions match. Otherwise I would have updated it too.